README.md

```diff
@@ -107,6 +106,153 @@ print(myShoppingList)
 // prints "["Almond Butter", "Jelly", "Bread", "Strawberries", "Bananas"]"
 ```
 
-* Challenge the student to write a function that takes in an `Array` as an argument, the `Array`'s type is `[String]`, where they are asked to grab the first element in the array (at index 0) and see if it's value is equal to "Bread". If so, return `true`.. if not, return `false`.
+- Challenge the student to write a function that takes in an `Array` as an argument, the `Array`'s type is `[String]`, where they are asked to grab the first element in the array (at index 0) and see if it's value is equal to "Bread". If so, return `true`.. if not, return `false`.
+
+--------------------------------------------------------------------------------
+
+## Arrays
+
+So far you've used variables to store `String` and `Int` values. For example, you could create a variable to represent an ingredient:
+
+```swift
+let ingredient = "Jalapeno"
+```
+
+Variables only allow you to store one item of information at a time. What if we were making a shopping list and needed all the ingredients wanted to make a sandwich?
+
+You could create multiple variables to represent the ingredients:
+
+```swift
+let ingredient1 =  "Bread"
+let ingredient2 = "Butter"
+let ingredient3 = "Cheese"
+let ingredient4 = "Lettuce"
+let ingredient5 = "Tomatoes"
+```
+
+And then write code that passes these variables one at a time. This will get messy quickly. What if you forget an ingredient or need to add a new one? In the real world, you would write down all your ingredients on one shopping list. With Swift, like most programming languages, you achieve the same using an _array_. An array allows a variable to contain a _collection_ of values of the same type.
+
```

Comment:
This is awesome, thanks! Great job encapsulating the idea of an array in a story/metaphor.